src/domain/service_domain_db.erl

```diff
@@ -73,18 +73,34 @@ start_link() ->
 enabled() ->
     mongoose_service:is_loaded(?MODULE).
 
+all_members() ->
+    pg:get_members(?SCOPE, ?GROUP).
+
 force_check_for_updates() ->
     %% Send a broadcast message.
-    case pg:get_members(?SCOPE, ?GROUP) of
+    case all_members() of
         [_|_] = Pids ->
             [Pid ! check_for_updates || Pid <- Pids],
             ok;
         _ ->
             ok
     end.
 
+%% Ensure that all pending check_for_updates messages are received by this node,
+%% even if they are sent by other nodes in the cluster.
+%% We have to do an RPC to ensure there is nothing pending in the distributed communication buffers.
+%% Used in tests.
 sync_local() ->
-    gen_server:call(?MODULE, ping).
+    LocalPid = whereis(?MODULE),
+    true = is_pid(LocalPid),
+    Nodes = [node(Pid) || Pid <- all_members()],
+    %% Ping from all nodes in the cluster
+    [pong = rpc:call(Node, ?MODULE, ping, [LocalPid]) || Node <- Nodes],
```

Comment:
Why not `erpc:multical/4` instead maybe? Just a suggestion, otherwise I can merge as is.

Reply:
just to keep it simple. 
It is basically 2 nodes in the tests anyway.
And this function is used only in tests :D